src/main/kotlin/io/snyk/plugin/services/download/CliDownloaderErrorHandler.kt

```diff
@@ -14,16 +16,30 @@ class CliDownloaderErrorHandler {
         SnykBalloonNotificationHelper.showError(message, project,
             NotificationAction.createSimple("Retry CLI download") {
                 project.getService(SnykCliDownloaderService::class.java).downloadLatestRelease(indicator, project)
+                runBackgroundableTask("Retry Snyk CLI Download", project, false) {
+                    project.getService(SnykCliDownloaderService::class.java).downloadLatestRelease(it, project)
```

Comment:
Kptlin has nice inline fun for that: project.service<T>()

Reply:
we are talking about this one: https://plugins.jetbrains.com/docs/intellij/plugin-services.html#retrieving-a-service